packages/stores/src/account/index.ts

```diff
@@ -815,6 +815,91 @@ export class OsmosisAccountImpl {
     );
   }
 
+  async sendAddToConcentratedLiquidityPositionMsg(
+    positionId: string,
+    amount0: string,
+    amount1: string,
+    maxSlippage = "2.5",
```

Comment:
just curious where this number comes from? We should probably put it in a constant with a description

Reply:
It's arbitrary, a normal slippage % value. I can add docs to fn sig.

Reply:
More background; wherever you see slippage being specified, typically token amounts are being transformed from one form to another (like L values with concentrated liquidity, or gamm shares for balancer and stable pools, or in and out of a constantly changing AMM pool). Slippage generally refers to the changes to chain state between signing the tx and the tx being committed in the block.

Here, liquidity values could change between signing and the position being created. So we subtract 2.5% from the amount calculated and use that as minimum amounts the user is willing to accept as addition to the position. It's a user protection mechanism.

Reply:
Makes sense. I still think we should put in a constant though